src/MOLFiniteDifference/MOL_discretization.jl

```diff
@@ -1,4 +1,7 @@
 using ModelingToolkit: operation, istree, arguments
+#using SymbolicUtils
```

Comment:
?

Reply:
Thanks for pointing this out to me, I just removed those comments.